gammapy/irf/psf_table.py

```diff
@@ -586,7 +586,8 @@ def kernels(self, cube, **kwargs):
             energy_band = Quantity([emin, emax])
             try:
                 psf = self.table_psf_in_energy_band(energy_band, **kwargs)
-                kernel = psf.kernel(cube.sky_image_ref)
+                rad_max = psf.containment_radius(0.99)
```

Comment:
For consistency it might be nice to introduce the `rad_max` argument for the `.kernels()` method as well. This would also resolve the TODO comment above. If you find time to do it: great! If not I can open an issue...

Reply:
I though about this as well, but the issue is that you probably want an energy dependent kernel size, don't you? of course you could put an array for ``rad_max`` but then one would have to compute the PSF in energy bands beforehand. This will then be done again in the ``kernels`` method ...

Reply:
On the other hand, if you choose a large enough kernel size it shouldn't really matter, I'm not super familiar with all with convolution stuff